### ISSUE TITLE
React DevTools Node Picker should have a loading... tooltip

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#5535]
It takes ~2 seconds for the Node picker to start. We should show a Loading... tooltip while you hover in the canvas (similar to the elements panel).

**Comment by bvaughn** [comment]
I believe the amount of time it takes for the tool to load depends on the profile. (I recorded a small project to test this, and it seems to load immediately.)

I recorded a larger profile, using Facebook.com, and that behaved a little strangely. Seems like no matter what I do on that one, after clicking the React DevTools "inspect" button, the tooltip shows "_Add comment_". (I also eventually see a bridge protocol mismatch error for that Replay profile, although maybe that's expected.)

**Comment by bvaughn** [comment]
I think the in between time (when "_Add comment_" is shown instead of the picker) is due to these async functions:
https://github.com/RecordReplay/devtools/blob/27fb800ea62f99d537e669f7a4141be12e1fe655/src/ui/components/SecondaryToolbox/ReactDevTools.tsx#L122-L125

Once `setIsNodePickerActive` is called (via this call):
https://github.com/RecordReplay/devtools/blob/27fb800ea62f99d537e669f7a4141be12e1fe655/src/ui/components/SecondaryToolbox/ReactDevTools.tsx#L134

Then the `<Video>` player properly suppresses that tooltip:
https://github.com/RecordReplay/devtools/blob/27fb800ea62f99d537e669f7a4141be12e1fe655/src/ui/components/Video.tsx#L81

That's not quite the same as the "_Loading..._" tooltip but it's related.

**Comment by bvaughn** [comment]
This is a good example of functionality that should be encapsulated within React DevTools core (and exposed via some nice public API). 😄 
https://github.com/RecordReplay/devtools/blob/27fb800ea62f99d537e669f7a4141be12e1fe655/src/ui/components/SecondaryToolbox/ReactDevTools.tsx#L177-L195

cc @lunaruan @mondaychen (for future input into the v5 rewrite)

**Comment by bvaughn** [comment]
Back on this issue again now after a bit of a detour.

I think the fix is going to require a little more code than I thought it would at first. Basically the "loading..." tooltip is managed by the `CommentTool` which isn't shown if you aren't _commenting_ (which you aren't if you're using one the element pickers).

I think the fix here then has a few steps:
1. Add an additional state to the Redux store (e.g. `isNodePickerInitializing`).
2. Update our person-in-the-middle `Wall` to set this state while it's initializing the React picker.
3. Update the `Video` component to respect this new state value.
4. Add a tooltip (since we can't just re-use the comment on). Maybe we _can_ add a reusable one though (and maybe even update the `CommentTool` to use it?) since there's some non-trivial imperative action required to track the cursor.

**Comment by lunaruan** [comment]
> This is a good example of functionality that should be encapsulated within React DevTools core (and exposed via some nice public API). 😄

Yeah, definitely! We're probably not going to work on React DevTools v5 until late next half (at the earliest), but we'll be sure to loop Replay in when we start designing the API to make sure it works for your use cases.
